srai/utils/merge.py

```diff
@@ -0,0 +1,42 @@
+"""Utility function for merging Shapely polygons."""
+
+from typing import List, Union
+
+import geopandas as gpd
+from shapely.geometry import MultiPolygon, Polygon
+
+
+def merge_polygons(polygons: List[Union[Polygon, MultiPolygon]]) -> MultiPolygon:
```

Comment:
Not sure if `merge` is a right naming, since all I'm doing is joining polygons into a single multipolygon and any overlap might result in a error (I didn't check).
Maybe someone will have a better naming convention. I specifically wrote that polygons are expected to be disjointed.
I need it to take all regions on the Earth and join it to check if it's equal to a full bounding box (-180, -90, 180, 90).

Reply:
If it is only used by us, we could treat it as an internal function and it can be specific for disjointed polygons. Then maybe rename it to `_megre_polygons_disjointed`. WDYT?

Reply:
I guess `_merge_disjointed_polygons` might be viable for now, but I can't guarantee that this function won't be used somewhere externally.

Reply:
Maybe `group` instead of `merge` would be better, but in geo space, if someone wants to create a single polygon without overlaps then they use the `union` word, so maybe `merge` isn't so bad.

Reply:
Changed to `_merge_disjointed_polygons`